test/integration/stocks.test.ts

```diff
@@ -0,0 +1,109 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+
+import app from "../../server";
+import authService from "../../services/authService";
+import addUser from "../utils/addUser";
+import cleanDb from "../utils/cleanDb";
+import stocks from "../../models/stocks";
+import sinon from "sinon";
+import config from "config";
+
+const cookieName: string = config.get("userToken.cookieName");
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe("GET /stocks/:userId", function () {
+  let jwt: string;
+  let userId: string;
+
+  beforeEach(async function () {
+    userId = await addUser();
+    jwt = authService.generateAuthToken({ userId });
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+    sinon.restore();
+  });
+
+  it("Should return user stocks when stocks are available", function (done) {
+    const userStocks = [{id: "5YGjUSW1SinwCNfuLXOO", userId: "DHLG3gYMTtMenj6lciWz", stockId: "s2eYDswDUAoQxwAhh07f", stockName: "EURO", quantity: 1, orderValue: 150, initialStockValue: 150}];
+
+    sinon.stub(stocks, "fetchUserStocks").resolves(userStocks);
```

Comment:
Why is this stubed?
We have a db emulator in place, so we should not be mocking it

Reply:
done